install.sh

```diff
@@ -211,35 +214,37 @@ done
 
 
 function installPrerequisities {
-  echo -e "${INFO} Configuring docker-ce and docker-compose"
-  echo -e "${INFO} Checking curl"
-  apt-get update -qq
-  apt-get install curl -qq -y
-
-  if test -f /usr/bin/dockerd; then
-    dockerdVersion=$(/usr/bin/dockerd --version)
-    echo -e "${INFO} $dockerdVersion already installed, skipping..."
-  else
-    echo -e "${INFO} Installing docker-ce"
-    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
-    apt-get install software-properties-common
-    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  if [[ "${__SKIP_DEP}" != 'true' ]]; then
+    echo -e "${INFO} Configuring docker-ce and docker-compose"
+    echo -e "${INFO} Checking curl"
     apt-get update -qq
-    apt-get install -qq -y $dockerInstallVersion
+    apt-get install curl -qq -y
 
-    if [ $skipswarm -eq 0 ]; then
-      echo -e "${INFO} Initializing docker in swarm mode"
-      docker swarm init
+    if test -f /usr/bin/dockerd; then
+      dockerdVersion=$(/usr/bin/dockerd --version)
+      echo -e "${INFO} $dockerdVersion already installed, skipping..."
+    else
+      echo -e "${INFO} Installing docker-ce"
+      curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+      apt-get install software-properties-common
+      add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+      apt-get update -qq
+      apt-get install -qq -y $dockerInstallVersion
+
+      if [ $skipswarm -eq 0 ]; then
```

Comment:
just wondering, why is this different from similar codnition above

[[ "${__SKIP_DEP}" != 'true' ]]

and where is __NO_SWARM used